_storage/write-to-read.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: storage
+permalink: /:collection/:path.html
+---
+# Storage write and read
+
+Once a user authenticates and a DApp obtains authentication, the application interacts with Gai through the blockstack.js library. There are two simple methods for working with data in Gaia hub the `putFile()` and `getFile()` methods. This section goes into greater detail about the methods, how they interact with a hub, and how to use them.
```

Comment:
"Gaia" not "Gai"

Reply:
Should we link "blockstack.js library" to the GitHub repo or other docs about it?

Reply:
"There are two simple methods for working with data in a Gaia hub: the `putFile()` and `getFile()` methods."

Reply:
blockstack.js in the Reference section.